### DISCUSSION TITLE
关于Shm的问题

### DISCUSSION THREAD

**Discussion by Ye980226** [wondertrader/wondertrader#238] [想法/Ideas]
比如说针对股票高频的场景，可能会有十个核去做核心的交易逻辑，一个核处理行情和发单，所以只希望连一个td，以及一个md，现在md可以通过ParserShm解决，不知道您目前有没有支持TraderShm的想法

**Comment by wondertrader** [comment]
有的，但是不会开源了

**Reply by Ye980226** [reply]
哦哦，谢谢，那您能不能大概说一下设计的架构，是另外起一个独立的进程来管理TraderShm吗，还是会跟数据写共享内存混合在一起
